### PR TITLE
Created a unique names test (Issue #234)

### PR DESCRIPTION
**Pull request by MoromugHSE**
Created a test suggested by issue #234, checking that we have no conflicting metric names.

### CODE REVIEW

File: tests/steps/test-unique.sh [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        Foo(String zz) {
+            this.z = zz;
+        }
+        private final boolean boom() { return true; }
+    }
+EOT
+    $("${LOCAL}/steps/measure-file.sh" "${java}" "${temp}/m1")
+    set -x
+    all=$(find "${temp}" -name 'm1.*' -type f -exec basename {} \; | sort)
+    expected=52
```
Comment (yegor256) [inline tests/steps/test-unique.sh:46]: @MoromugHSE it's a typo, I believe? The `expected` should be counted, not be a constant?

**Comment by yegor256** [comment]
@MoromugHSE try to run `make lint` on your machine

**Comment by yegor256** [comment]
@MoromugHSE did you run `make test` locally?

**Comment by MoromugHSE** [comment]
> @MoromugHSE did you run `make test` locally?

Has just run it again (in docker) -- all tests pass locally, yet test-unique.sh is somehow not executable for the workflows.

**Comment by yegor256** [comment]
@MoromugHSE please, read this: https://www.yegor256.com/2024/04/01/ping-me-please.html I keep missing your messages, that's why delays

**Comment by MoromugHSE** [comment]
@yegor256 finally found the command for GitHub to acknowledge the exec permission -- rather surprised how much time did it take to even think about looking for it...

**Comment by yegor256** [comment]
@MoromugHSE thanks!

**Pull request merged**